tests/chocolatey-tests/Hooks.Tests.ps1

```diff
@@ -0,0 +1,93 @@
+Import-Module helpers/common-helpers

```

Comment:
Since this file explicitly test a specific feature, can you move this file into a `features` directory, since it doesn't follow the same conventions we already use by having the test file be `choco-` command name.

Reply:
I've moved this and a few other files around as it seemed to make sense.